sdks/python/apache_beam/ml/inference/pytorch_test.py

```diff
@@ -117,11 +135,104 @@ def test_inference_runner_multiple_tensor_features(self):
                      ('linear.bias', torch.Tensor([0.5]))]))
     model.eval()
 
+    inference_runner = PytorchInferenceRunner(torch.device('cpu'))
+    predictions = inference_runner.run_inference(examples, model)
+    for actual, expected in zip(predictions, expected_predictions):
+      self.assertEqual(actual, expected)
+
+  def test_inference_runner_kwargs(self):
```

Comment:
I suggest a short explanation at the head of these tests, since they're more complex than simple unit test.